config/default.ts

```diff
@@ -1,7 +1,14 @@
-export default ({ 
-    APIGenerated: {
-      port: process.env.PORT || 3000,
-      enforceWhitelist: process.env.WHITELIST || "FALSE",
-      mainnet: process.env.MAINNET || "FALSE",
-    }
-  } as ConfigType);
\ No newline at end of file
+export default {
+  sidechain: {
+    nodeUrl: process.env.CONTRACT_HOST || "http://localhost:8545",
+    address: process.env.CONTRACT_ADDRESS || "0x0000000000000000000000000000000000008888",
```

Comment:
I don't want to be PITA, but can we name this `accountIngressAddress`? :smile: because now it's a bit like what address?

Reply:
Of course we can :+1: . I'll change this then.